includes/callbacks.inc

```diff
@@ -90,20 +91,21 @@ function islandora_entities_autocomplete($type) {
     'fl' => array($mappings[$type]['title'], 'PID'),
```

Comment:
One other thing (haha sorry 😅) - the `fl` param isn't used in `internalSolrParams` and won't be set by `buildQuery()` - and by extension, `buildAndExecuteQuery()`. So in cases where an `fl` is required here, it's going to be missed unless we `buildQuery()`, explicitly set `solrParams['fl']`, and then `executeQuery()`. This happens in a few other places throughout here.